diagrams/onprem/certificates.py

```diff
@@ -0,0 +1,19 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
+
+from . import _OnPrem
+
+
+class _Certificates(_OnPrem):
+    _type = "certificates"
+    _icon_dir = "resources/onprem/certificates"
+
+
+class Certmanager(_Certificates):
```

Comment:
I request some changes similar to [this request change](https://github.com/mingrammer/diagrams/pull/260#discussion_r487851767).

It should be `class CertManager`.